Template/LO/SubProcesses/setscales.f

```diff
@@ -11,6 +11,7 @@ subroutine set_ren_scale(P,rscale)
 c
       include 'genps.inc'
       include 'nexternal.inc'
+      include 'vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc